src/libmongoc/src/mongoc/mongoc-topology-background-monitoring.c

```diff
@@ -272,6 +275,10 @@ _mongoc_topology_background_monitoring_stop (mongoc_topology_t *topology)
    int i;
    bool is_srv_polling;
 
+   /* Test only assert. Is a noop unless -DENABLE_TEST_ASSERTIONS=ON is set
```

Comment:
I'd argue for moving this comment, (and the ones in mongoc-topology.c), to the definition of `MONGOC_TEST_ASSERT` in common-macros-private.h.